doc/api/util.md

```diff
@@ -1097,6 +1097,80 @@ console.log(values, positionals);
 // Prints: [Object: null prototype] { foo: true, bar: 'b' } []
 ```
 
+Detailed parse information is available for adding custom behaviours by
+specifying `tokens: true` in the configuration. The returned tokens have
+properties describing:
+
+* all tokens
+  * `kind` { string } One of 'option', 'positional', or 'option-terminator'.
+  * `index` { number } Index of element in `args` containing token.
+* option tokens
+  * `name` { string } Long name of option.
+  * `rawName` { string } How option used in args, like `-f` of `--foo`.
+  * `value` { string | undefined } Option value specified in args.
+    Undefined for boolean options.
+  * `inlineValue` { boolean | undefined } Whether option value specified inline,
+    like `--foo=bar`.
+* positional tokens
+  * `value` { string } Positional value (i.e. `args[index]`).
```

Comment:
`value` appears in this list twice; not sure what this second one is supposed to be.

Reply:
The second `value` is the only additional property for a `positional` token over the common properties. (Somewhat in line with @bcoe suggestion of class!)

The indentation and hierarchy is not very clear, especially in the raw form.

The "subtitle" could be more explicit? Say:

```
* positional tokens have `value` (in addition to `kind` and `index`)
  * `value` { string } Positional value (i.e. `args[index]`).
```

Reply:
Ohhhh got it. "Positional value" confused me; maybe "the value of the positional argument"?

Reply:
Updated